crates/bevy_scene/src/lib.rs

```diff
@@ -335,6 +338,9 @@ mod tests {
             .insert(&scene_handle, scene_1);
 
         app.update();
+        app.update();
```

Comment:
```suggestion
        // TODO: multiple updates to avoid debounced asset events. See comment on SceneSpawner::debounced_scene_asset_events
        app.update();
```